src/analyses/useAfterFree.ml

```diff
@@ -27,40 +28,61 @@ struct
   let get_current_threadid ctx =
     ctx.ask Queries.CurrentThreadId
 
+  let get_joined_threads ctx =
+    ctx.ask Queries.MustJoinedThreads
+
+  let warn_about_deactivated_thread_joins () =
+    if not @@ List.mem "threadJoins" @@ get_string_list "ana.activated" then
+      M.warn "Running without thread joins analysis. Multi-threaded UAF detection might be imprecise!"
```

Comment:
This "warning" is currently causing some CI failures. I think we don't need this because that applies generally to many of our (multi-threaded) analyses.